kpet/schema.py

```diff
@@ -396,13 +396,28 @@ class List(Node):
     """
     List schema, with every element matching a single specified schema.
     """
-    def __init__(self, element_schema):
+    def __init__(self, element_schema, min_len=0):
+        """
+        The constructor needs an object of the type the list items must be.
+        It also supports an optional "min_len" parameter, to force the list
+        to contain at least "min_len" elements.
+        """
```

Comment:
Could you please follow the documentation format used for functions through the code? I.e. use imperative mood and describe the arguments separately under "Args:"? The style is similar to the [Google Python Style](https://google.github.io/styleguide/pyguide.html#383-functions-and-methods), except using the imperative, not descriptive language.

Reply:
Oops, didn't realise there was a documentation convention. Fixed now.

Are we using the whole style guide, or just for the documentation bit? Or, I guess the coding style is set by flake8?

Reply:
Thanks!

I started with "google style guide with exceptions", but right now it's mostly what flake8 is dictating. Right now nobody seems interested (except me maybe) in codifying and enforcing a coding style otherwise.